src/adapter/src/notice.rs

```diff
@@ -342,6 +346,9 @@ impl fmt::Display for AdapterNotice {
             AdapterNotice::WebhookSourceCreated { url } => {
                 write!(f, "URL to POST data is '{url}'")
             }
+            AdapterNotice::PerReplicaLogRead { log_names } => {
+                write!(f, "Queried per-replica objects: {}. Unlike other objects in Materialize, these results depend on the current values of the `CLUSTER` and `REPLICA` session variables.", log_names.join(", "))
```

Comment:
I think we can improve this message:
* Change "per-replica" objects to introspection relations, to be consistent with our docs.
* "Unlike other objects in Materialize, these results depend" is confusingly worded, because it sounds like results are objects.
* The session variables are called `cluster` and `cluster_replica`.